kernel/src/main/scala/cats/kernel/instances/StaticMethods.scala

```diff
@@ -17,6 +19,22 @@ object StaticMethods extends cats.kernel.compat.HashCompat {
     def iterator: Iterator[(K, V)] = m.iterator
   }
 
+  /**
+   * When you "own" this m, and will not mutate it again, this
+   * is safe to call. It is unsafe to call this, then mutate
+   * the original collection.
+   *
+   * You are giving up ownership when calling this method
+   */
+  def wrapMutableIndexedSeq[A](m: mutable.IndexedSeq[A]): ImIndexedSeq[A] =
+    new WrappedIndexedSeq(m)
+
+  private[kernel] class WrappedIndexedSeq[A](m: mutable.IndexedSeq[A]) extends ImIndexedSeq[A] {
```

Comment:
I can never remember exactly how this works inside objects, but is there any reason to make this package-private (which is just public in the JVM API) instead of just private (which I think should be properly JVM-private)?

Reply:
not that I know of, I just copied the idiom below. Should I fix that up?